crates/network-monitor/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "network-monitor"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+devolutions-agent-shared = { path = "../devolutions-agent-shared" }
+network-scanner = { path = "../network-scanner" }
+network-scanner-net = { path = "../network-scanner-net" }
+serde = { version = "1", features = ["derive"] }
+camino = { version = "1", features = ["serde1"] }
+serde_json = "1"
+thiserror = "1"
+chrono = { version = "0.4", features = ["serde"] }
+tokio = { version = "1.45", features= ["full"] }
+tokio-util = { version = "0.7", features = ["full"] }
```

Comment:
issue: Do not use `"full"`, especially in libraries, this brings in new dependencies and code we don’t need, including some big ones (e.g.: slab).